### PR TITLE
etcher: 1.5.86 -> 1.5.117

### PR DESCRIPTION
**Pull request by yorickvP**
###### Motivation for this change

Update

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120334` at 2a619871 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>etcher</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/patch-phase.md">patch-phase</a>

`patchPhase` should not be overridden, use `postPatch` instead.

Near pkgs/tools/misc/etcher/default.nix:47:3:
```
   |
47 |   patchPhase = ''
   |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `polkit`.
Near pkgs/tools/misc/etcher/default.nix:5:3:
```
  |
5 | , polkit
  |   ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/etcher/default.nix [hunk 1]
```diff
@@ -46,14 +45,13 @@ stdenv.mkDerivation rec {
...
-    sed -i "s|/usr/bin/pkexec|/usr/bin/pkexec', '/run/wrappers/bin/pkexec|" tmp/node_modules/sudo-prompt/index.js
-    sed -i 's|/bin/bash|${bash}/bin/bash|' tmp/node_modules/sudo-prompt/index.js
-    sed -i "s|'lsblk'|'${util-linux}/bin/lsblk'|" tmp/node_modules/drivelist/js/lsblk/index.js
-    sed -i "s|process.resourcesPath|'$out/share/${pname}/resources/'|" tmp/generated/gui.js
-    ${nodePackages.asar}/bin/asar pack tmp opt/balenaEtcher/resources/app.asar
-    rm -rf tmp
+    sed -i 's|/usr/bin/pkexec|/usr/bin/pkexec", "/run/wrappers/bin/pkexec|' generated/gui.js
+    sed -i 's|/bin/bash|${bash}/bin/bash|' generated/gui.js
+    sed -i 's|"lsblk"|"${util-linux}/bin/lsblk"|' generated/gui.js
+    sed -i "s|process.resourcesPath|'$out/share/${pname}/resources/'|" generated/gui.js
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/etcher/default.nix:53]: Please use one substituteInPlace instead.

**Comment by lukegb** [comment]
Bump. PTAL at the outstanding review comments.

**Comment by yorickvP** [comment]
Fixed.

**Pull request merged**